Signal/src/call/SignalCall.swift

```diff
@@ -58,6 +58,9 @@ enum CallState: String {
         return lhs.localId == rhs.localId
     }
 
+    static func newCallSignalingId() -> UInt64 {
```

Comment:
I've exposed this as a function so that tests and debugging logic can generate suitable "call signaling ids."